keps/sig-node/4540-strict-cpu-reservation/README.md

```diff
@@ -333,33 +328,32 @@ No changes needed.
 
 ### Feature Enablement and Rollback
 
-The `/var/lib/kubelet/cpu\_manager\_state` needs to be removed when enabling or disabling the feature.
+The `/var/lib/kubelet/cpu_manager_state` needs to be removed when enabling or disabling the feature.
 
 ###### How can this feature be enabled / disabled in a live cluster?
 
 - [X] Feature gate (also fill in values in `kep.yaml`)
-  - Feature gate name: `CPUManagerPolicyAlphaOptions`
+  - Feature gate name: `CPUManagerPolicyBetaOptions`
```

Comment:
Honestly, I'm a bit worried about the feature gate renaming here. Was is tested how this works with version emulation?

Reply:
If I follow the current flow, one option is under one gate per https://github.com/kubernetes/kubernetes/blob/master/pkg/kubelet/cm/cpumanager/policy_options.go#L40

However, I don't see issue to support one option under both Alpha and Beta gates. I am fine either way.

@swatisehgal what do you think?